helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/listener/HelpDeskRoomEventListener.java

```diff
@@ -115,8 +132,43 @@ public void onSymUserLeftRoom(SymUserLeftRoom symUserLeftRoom) {
     SymUser symUser = symUserLeftRoom.getAffectedUser();
     SymStream symStream = symUserLeftRoom.getStream();
 
-    if (isBotUser(symUser) && isAgentStreamId(symStream)) {
-      LOGGER.warn("Bot user was removed from the agent room");
+    if (!isAgentStreamId(symStream) && (symStream.getMembers().size() <= 1)) {
+      Ticket ticket = ticketClient.getTicketByServiceStreamId(symStream.getStreamId());
+
+      if (ticket != null) {
+        LOGGER.info("Only the bot was left in the ticket room. Reopening ticket in the Agent room");
+
+        // Update ticket to a state that it can be claimed again by another agent
+        ticket.setAgent(null);
+        ticket.setState(TicketClient.TicketStateType.UNSERVICED.toString());
+        ticketClient.updateTicket(ticket);
+
+        SymUser localUser = symphonyClient.getLocalUser();
+
+        // Retrieve the client question
+        String question = StringUtils.EMPTY;
+        try {
+          // FIXME Get the timestamp from the ticket
+          SymMessage clientMessage =
+              symphonyClientUtil.getSymMessageByStreamAndTimestamp(ticket.getClientStreamId(), 0L);
+          question = clientMessage.getMessageText();
+        } catch (MessagesException e) {
+          LOGGER.warn("The client question could not be retrieved.");
```

Comment:
Please upgrade this error level to ERROR and include the root cause as part of the logging.

```
LOGGER.error("The client question could not be retrieved.", e);
```

Reply:
Done